### ISSUE TITLE
sqlite warning "Table ... has column PRIMARY but struct has not related field"

### ISSUE THREAD

**Issue by russtone** [go-xorm/xorm#491]
Hello!
I have struct a struct with composite primary key:
```go
type TaskSolution struct {
	UID     string    `xorm:"notnull pk UUID 'uid'"`
	TID     string    `xorm:"notnull pk UUID 'tid'"`
	Created time.Time `xorm:"created"`
	Updated time.Time `xorm:"updated"`
}
```
And when I use sqlite3 I constantly get warning: `Table task_solution has column PRIMARY but struct has not related field`. 
What is the problem here?

**Comment by lunny** [comment]
I cannot reproduce your situation. This is my test code:
```Go
package main

import (
	"os"
	"time"

	"github.com/go-xorm/xorm"
	_ "github.com/mattn/go-sqlite3"
)

type TaskSolution struct {
	UID     string    `xorm:"notnull pk UUID 'uid'"`
	TID     string    `xorm:"notnull pk UUID 'tid'"`
	Created time.Time `xorm:"created"`
	Updated time.Time `xorm:"updated"`
}

func main() {
	os.Remove("./db")
	engine, err := xorm.NewEngine("sqlite3", "./db")
	if err != nil {
		panic(err)
	}
	engine.ShowSQL(true)

	err = engine.Sync2(new(TaskSolution))

	if err != nil {
		panic(err)
	}
}
```

**Comment by russtone** [comment]
It seems that warnings appears only if database is already exists.
I can reproduce it with this code:
```go
package main

import (
	"time"

	"github.com/go-xorm/xorm"
	_ "github.com/mattn/go-sqlite3"
)

type TaskSolution struct {
	UID     string    `xorm:"notnull pk UUID 'uid'"`
	TID     string    `xorm:"notnull pk UUID 'tid'"`
	Created time.Time `xorm:"created"`
	Updated time.Time `xorm:"updated"`
}

func main() {
	engine, err := xorm.NewEngine("sqlite3", "./db")
	if err != nil {
		panic(err)
	}
	engine.ShowSQL(true)

	err = engine.Sync2(new(TaskSolution))

	if err != nil {
		panic(err)
	}
}
```
After second run I got:

```
[xorm] [info]  2016/12/13 14:09:08.513900 [sql]SELECT name FROM sqlite_master WHERE type='table'
[xorm] [info]  2016/12/13 14:09:08.514864 [sql]SELECT sql FROM sqlite_master WHEREtype='table' and name = ?[task_solution]
[xorm] [info]  2016/12/13 14:09:08.515012 [sql]SELECT sql FROM sqlite_master WHEREtype='index' and tbl_name = ?[task_solution]
[xorm] [warn]  2016/12/13 14:09:08.515116 Table task_solution has column PRIMARY but struct has not related field
```

**Comment by lunny** [comment]
resolved by #606
